client/scripts/controllers/chain/ethereum/main.ts

```diff
@@ -38,10 +39,14 @@ class Ethereum extends IChainAdapter<EthereumCoin, EthereumAccount> {
     await this.accounts.init(this.chain);
     await this.chain.initEventLoop();
 
-    await this.webWallet.web3.givenProvider.on('accountsChanged', function (accounts) {
-      const updatedAddress = this.app.login.activeAddresses.find((addr) => addr.address === accounts[0])
-      selectLogin(updatedAddress);
-    });
+    if (this.webWallet) {
+      const app = this.app;
```

Comment:
why add this when you can call it directly below? `this.app.login.activeAddresses`